.github/scripts/trymerge.py

```diff
@@ -1511,6 +1521,16 @@ def handle_exception(e: Exception, title: str = "Merge failed") -> None:
         gh_post_pr_comment(org, project, args.pr_num, "Cross-repo ghstack merges are not supported", dry_run=args.dry_run)
         return
 
+    if not has_required_labels(pr):
+        gh_post_pr_comment(
+            org,
+            project,
+            args.pr_num,
+            "Can't merge without required labels, see CI job check_labels comment for details",
```

Comment:
As part of this flow, we'll want to remove the check_labels job as well. 

Can you replace this message with the text `check_labels.py` outputs:
https://github.com/pytorch/pytorch/blob/1facb4a561e89daf44876f98a62f6e9dd0f4332a/.github/scripts/check_labels.py#L21-L26

Reply:
Removed into `trymerge.py`